### PR TITLE
fix keywindow assignment

### PR DESCRIPTION
**Pull request by tomsemi**
reset keywindow after alert

### CODE REVIEW

**Comment by frenetisch-applaudierend** [comment]
Hi Tom, thanks for this fix, much appreciated 👍

File: Sources/FPSStatusBarViewController.swift [hunk 1]
```diff
@@ -86,7 +86,9 @@ extension FPSStatusBarViewController: FPSCounterDelegate {
     @objc func fpsCounter(_ counter: FPSCounter, didUpdateFramesPerSecond fps: Int) {
         let milliseconds = 1000 / max(fps, 1)
         self.label.text = "\(fps) FPS (\(milliseconds) milliseconds per frame)"
-
+        if FPSStatusBarViewController.statusBarWindow == UIApplication.shared.keyWindow {
```
Comment (frenetisch-applaudierend) [inline Sources/FPSStatusBarViewController.swift]: Could you please do `FPSStatusBarViewController.statusBarWindow.isKeyWindow` instead? This captures the intention more clearly.

Comment (tomsemi) [reply Sources/FPSStatusBarViewController.swift]: I've modified that，thx

**Comment by frenetisch-applaudierend** [comment]
Awesome, thanks 👍

**Pull request merged**